apps/web/src/services/administration/components/AppManagementFormModal.vue

```diff
@@ -111,13 +115,13 @@ const fetchListRoles = async (inputText: string) => {
     }
 
     try {
-        const response = await appPageStore.listRoles({
+        const { results } = await SpaceConnector.clientV2.identity.role.list<RoleListParameters, ListResponse<RoleModel>>({
             query: roleListApiQueryHelper.data,
         });
-        dropdownState.menuItems = response.map((role) => ({
+        dropdownState.menuItems = results?.map((role) => ({
```

Comment:
issue: `results` can be an empty object. I recommend adding defense code as shown below.
```suggestion
        dropdownState.menuItems = (results ?? []).map((role) => ({
```